src/LLVMDisassembler-Tests/LLVMDisassemblerTests.class.st

```diff
@@ -170,6 +170,15 @@ LLVMDisassemblerTests >> testDisassembleWithAlternatePrinterVariantReturnsAltern
 	self assert: instruction assembly equals: 'inc	ecx'
 ]
 
+{ #category : #tests }
+LLVMDisassemblerTests >> testInitializeOnlyOnce [
+
+	| dis n |
+	dis := LLVMDisassembler i386.
+	n := WeakRegistry default keys occurrencesOf: dis.
```

Comment:
Does this work on latest Pharo11?
Thinking it twice, it should because `WeakRegistry default` returns `FinalizationRegistry default`.

If we do the change below, this will stop working on Pharo < 11.

```suggestion
	n := FinalizationRegistry default keys occurrencesOf: dis.
```

So I think it's better to keep it this way for now.